Desktop/Observatory/observatory-server/server/api/user_registration/user_registration_controller.js

```diff
@@ -0,0 +1,90 @@
+import _ from 'lodash'
+import { handleError } from '../lib/helpers'
+import UserRegistration from './user_registration_model'
+
+/**
+* @api {get} /api/user_registration Index
+* @apiName index
+* @apiGroup UserRegistration
+* @apiDescription Get list of UserRegistrations for the requesting user
+* @apiPermission private
+* @apiSuccess {Collection} root Collection of all the user registration's.
+* @apiError (500) UnknownException Could not retrieve UserRegistration collection
+*/
+exports.index = async (req, res) => {
+  try {
+    const collection = await UserRegistration.find()
+    return res.json(200, collection).end()
+  } catch (err) {
+    return handleError(res, err)
+  }
+}
+
+/**
+* @api {get} /api/user_registration/:userid Show
+* @apiName show
+* @apiGroup UserRegistration
+* @apiDescription Show an individual user registration
+* @apiPermission private
+* @apiSuccess {Model} root A single UserRegistration model
+* @apiError (500) UnknownException Could not retrieve UserRegistration model
+*/
+exports.show = async (req, res) => {
+  try {
+    const model = await UserRegistration.findByUserId(req.params.userid)
```

Comment:
Little error here - should be:

```js
const model = await UserRegistration.findById(req.params.id)
```

Instead of `UserRegistration.findByUserId(...)`